### PR TITLE
Hide "[#9592]Searching Message History Coming Soon" On Chat Search And "Blocked Users" When Empty

### LINKED ISSUE #9592: Hide "searching message history coming soon" on chat search and "blocked users" when empty
# Problem

1. Let's hide this second section. It's confusing & message history is also not coming soon. 

image

2. Hide `Blocked users` until I have actually populated this list. No sense in showing 0 blocked users under contacts.

image

## Acceptance Criteria

- No more `Messages` section when you use the chat search bar.
- No `Blocked users` unless blocked users > 0.

### PR DESCRIPTION
**Pull request by tbenr**
fixes #9592

status: ready

### CODE REVIEW

**Review by vkjr — APPROVED** [review]

File: src/status_im/ui/screens/contacts_list/views.cljs [hunk 1]
```diff
@@ -21,14 +21,15 @@
...
      [react/scroll-view {:flex 1}
-      [list.views/big-list-item
-       {:style               {:margin-vertical 16}
-        :text                (i18n/label :t/blocked-users)
-        :icon                :main-icons/cancel
-        :icon-color          colors/red
-        :accessibility-label :blocked-users-list-button
-        :accessory-value     blocked-contacts-count
-        :action-fn           #(re-frame/dispatch [:navigate-to :blocked-users-list])}]
+      (when (> blocked-contacts-count 0)
```
Comment (yenda) [inline src/status_im/ui/screens/contacts_list/views.cljs]: replace with `pos?`

**Review by yenda — CHANGES_REQUESTED** [review]

File: src/status_im/ui/screens/contacts_list/views.cljs (hunk 1 above)
Comment (yenda) [inline src/status_im/ui/screens/contacts_list/views.cljs]:
```suggestion
      (when (pos? blocked-contacts-count)
```

Comment (tbenr) [reply src/status_im/ui/screens/contacts_list/views.cljs]: @yenda done

**Review by yenda — APPROVED** [review]

**Comment by churik** [comment]
Tested on IOS 13 (search and blocked users) and on Android 10 (only blocked users, search currently is not working on Android).
Looks good to me, thank you for contribution @tbenr !

**Pull request merged**